canopy/app/src/index.js

```diff
@@ -16,7 +16,25 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+
+import App from 'App';
+import rootReducer from 'reducers';
+
 import './index.css';
-import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const store = createStore(
+  rootReducer,
+
+  // This enables the use of redux devtools in the browser
+  // eslint-disable-next-line no-underscore-dangle
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
This should probably be behind a macro so it wouldn't get compiled into a production build.